### DISCUSSION TITLE
Simple Get not working, perhaps due to not using https?

### DISCUSSION THREAD

**Discussion by swimleftproducts** [Mair/esp32-course#76] [Course Qeustions] [answered]
I am working through the simple get example. I believe this is due to using http and not https. The output I receive from the esp32 is shown below. Using Curl confirmed what I was seeing in the terminal from the esp32. I see that we cover https next. I am mostly just leaving this post in case someone is scratching their head over this.

```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>301 Moved Permanently</title>
</head><body>
<h1>Moved Permanently</h1>
<p>The document has moved <a href="https://quotes.rest/qod">here</a>.</p>
</body></html>
```

E (1764) esp-tls-mbedtls: No server verification option set in esp_tls_cfg_t structure. Check esp_tls API reference
E (1764) esp-tls-mbedtls: Failed to set client configurations, returned [0x8017] (ESP_ERR_MBEDTLS_SSL_SETUP_FAILED)
E (1774) esp-tls: create_ssl_handle failed
E (1784) esp-tls: Failed to open new connection
E (1784) TRANSPORT_BASE: Failed to open a new connection
E (1794) HTTP_CLIENT: Connection failed, sock < 0
E (1804) WIFI CLIENT: http failed ESP_ERR_HTTP_CONNECT

**Comment by Mair** [comment, marked as answer]
@swimleftproducts that's correct. the nice thing about this endpoint is that it allows http as well as https

https://quotes.rest/qod
http://quotes.rest/qod

just remember that to get JSON you need to set the content type to application/json. Viewing these url's in the browser will render XML
